de-lib/src/main/java/org/iplantc/de/communities/client/presenter/ManageCommunitiesPresenterImpl.java

```diff
@@ -383,6 +397,11 @@ public void onFailure(Throwable caught) {
 
             @Override
             public void onSuccess(List<UpdateMemberResult> result) {
+                List<String> failedUsers = getFailedMemberUpdateNames(result);
+
+                if (failedUsers.size() > 0) {
+                    announcer.schedule(new ErrorAnnouncementConfig(appearance.joinCommunityFailure()));
+                }
                 callback.onSuccess(null);
```

Comment:
Should we call error callback instead when the request fails for all the users ? 🤔

Reply:
Ooh, really good catch - I think in all these cases, it actually makes more sense to call the `errorCallback`.  I'll refactor